### DISCUSSION TITLE
disable 2FA for my account

### DISCUSSION THREAD

**Discussion by muhammad-salem** [community/community#118787] [Other Feature Feedback, Questions, & Ideas]
i want to DISABLE 2FA  

![Screenshot from 2024-04-12 13-14-02](https://github.com/community/community/assets/7586302/14a3b3ec-7fe8-4122-969f-f2a4fd0e0e00)

for more information. i deleted the only organization that  i was contributing to.

**Comment by dorathoto** [comment]
Easy, but not recommended to disable 2-factor authentication.

[https://docs.github.com/en/enterprise-server@3.10/authentication/securing-your-account-with-two-factor-authentication-2fa/disabling-two-factor-authentication-for-your-personal- account](https://docs.github.com/en/enterprise-server@3.10/authentication/securing-your-account-with-two-factor-authentication-2fa/disabling-two-factor-authentication-for-your-personal- account)


### If you disable two-factor authentication for your personal account, you may lose access to organizations you belong to.

**Reply by muhammad-salem** [reply]
[Disabling two-factor authentication for your personal account](https://docs.github.com/en/enterprise-server@3.10/authentication/securing-your-account-with-two-factor-authentication-2fa/disabling-two-factor-authentication-for-your-personal-account) 

this docs page is useless 

![Screenshot from 2024-04-12 13-14-02](https://github.com/community/community/assets/7586302/14a3b3ec-7fe8-4122-969f-f2a4fd0e0e00)


for more information. i deleted the only organization that  i was contributing to.

**Comment by LiteBrite82** [comment]
Thanks for posting in the GitHub Community, @muhammad-salem !

We're happy you're here. You are more likely to get a useful response if you are posting your question in the applicable category, the Discussions category is solely related to conversations around the [GitHub product Discussions](https://docs.github.com/en/discussions/collaborating-with-your-community-using-discussions/about-discussions). This question should be in the General category. I've gone ahead and moved it for you. Good luck!

**Reply by muhammad-salem** [reply]
Thanks 👍
